components/MainNavBar.vue

```diff
@@ -222,6 +222,12 @@ export default {
             },
           ],
         },
+        {
+          name: "Memorials & Sites of Interest",
```

Comment:
Sorry I realised this ampersand probably needs encoding as well (any any others in the nav)

Reply:
doesn't need done